EKF/ekf_helper.cpp

```diff
@@ -79,7 +79,7 @@ void Ekf::resetHorizontalVelocityToOpticalFlow()
 	// calculate absolute distance from focal point to centre of frame assuming a flat earth
 	const float range = heightAboveGndEst / _range_sensor.getCosTilt();
 
-	if ((range - _params.rng_gnd_clearance) > 0.3f && _flow_sample_delayed.dt > 0.05f) {
+	if ((range - _params.rng_gnd_clearance) > 0.3f && _flow_sample_delayed.dt < 0.05f) {
```

Comment:
Was this always like this? So we reset always to 0?
Is this test really necessary? If yes, shouldn't we use it also for the normal fusion?

Reply:
Yes, apparently this never worked :D https://github.com/PX4/PX4-ECL/pull/74/files#diff-141e70f9277c0572024e165ddc7b56fffd1b604a06d9239c7410015f3a9ad0c7R89

Reply:
> Is this test really necessary? If yes, shouldn't we use it also for the normal fusion?

I would say that it is especially important during initialization to avoid resetting on a bad sample, but yes, this is something that can be done during normal fusion too.